openlibrary/components/AuthorIdentifiers.vue

```diff
@@ -0,0 +1,135 @@
+<template>
+  <table>
+    <tr>
+      <th>
+        <select class="form-control" v-model="selectedIdentifier" name="name">
+          <option disabled value="">Select one</option>
+          <option v-for="idConfig in identifierConfigsByKey" :key="idConfig.name" :value="idConfig.name">
+            {{idConfig.label}}
+          </option>
+        </select>
+      </th>
+      <th>
+        <input class="form-control" type="text" name="value" id="id-value" v-model="inputValue" @keyup.enter=setIdentifier>
+      </th>
+      <th>
+        <button class="form-control" name="set" :disabled="!setButtonEnabled" @click=setIdentifier>Set</button>
+      </th>
+    </tr>
+    <tr v-for="(value, name) in assignedIdentifiers" :key="name">
```

Comment:
See https://github.com/internetarchive/openlibrary/pull/5093#discussion_r633808865

```suggestion
    <tr v-for="(value, name) in assignedIdentifiers" :key="name" v-if="value">
```